Doc/reference/simple_stmts.rst

```diff
@@ -335,15 +335,20 @@ statement, of a variable or attribute annotation and an optional assignment stat
 The difference from normal :ref:`assignment` is that only single target is allowed.
 
 For simple names as assignment targets, if in class or module scope,
-the annotations are evaluated and stored in a special class or module
+the annotations are available in a special class or module
 attribute :attr:`__annotations__`
-that is a dictionary mapping from variable names (mangled if private) to
-evaluated annotations. This attribute is writable and is automatically
+that is a dictionary mapping from variable names (mangled if private) to the
+annotations.  This attribute is writable and is automatically
 created at the start of class or module body execution, if annotations
-are found statically.
+are found statically.  If the ``annotations`` import from :mod:`__future__` is used
+(i.e. :pep:`563` is enabled), the annotations are stored as string values in
```

Comment:
Ideally we should link to the documentation for the `__future__` statement, not to the PEP directly. A PEP is a change proposal, not live documentation.